gui/storage/forms.py

```diff
@@ -2397,11 +2397,11 @@ class VolumeExport(Form):
 
     def __init__(self, *args, **kwargs):
         self.instance = kwargs.pop('instance', None)
-        services = kwargs.pop('services', {})
+        attachments = kwargs.pop('attachments', [])
         super(VolumeExport, self).__init__(*args, **kwargs)
-        if list(services.keys()):
+        if attachments:
             self.fields['cascade'] = forms.BooleanField(
-                initial=True,
+                initial=False,
```

Comment:
Because it's more destructive now. Also new False is old True.

Reply:
More destructive how? We already did delete the shares before. Or do you mean because we also do for periodic snapshot task?

If thats the only case I think we are fine keeping the True or people will remain with the old shares.
Note that I see exporting a pool without destroying as an edge case, most people will destroy if they are at that point.